src/app/inputs/inputs.component.ts

```diff
@@ -6,5 +6,44 @@ import { Component } from '@angular/core';
   styleUrls: ['./inputs.component.css']
 })
 export class InputsComponent {
+  
 
+  addTask() {
+    console.log("Agregar tarea")
+    const inputTitle = document.getElementById('input-ul') as HTMLInputElement | null;
+
+    const taskTitle = inputTitle?.value;
+    console.log(taskTitle)
+    
+
+    const inputDesc = document.getElementById('desc-ul') as HTMLInputElement | null;
+
+    const taskDesc = inputDesc?.value;
+    console.log(taskDesc)
+
+  }
+
+  async getCats() {
+    const response = await fetch("https://catfact.ninja/fact");
+    const jsonData = await response.json();
+    return jsonData;
+  }
+
+  
+
+  seeCat(cat: { fact: any; }){
+    console.log(cat.fact)
+}
+
+  addCatsk(){
+    let json=this.getCats();
+    console.log(json.then(this.seeCat))
+  }
+
+  addLink()
```

Comment:
I would rename the function to `routeToVideo()`, since we are not adding anything really